README.md

```diff
@@ -1,4 +1,36 @@
 # Code Climate PHP Mess Detector (PHPMD) Engine
 
-`codeclimate-phpmd` is a Code Climate Engine that wraps the PHP Mess
-Detector (PHPMD) static analysis tool.
+`codeclimate-phpmd` is a Code Climate Engine that wraps the PHP Mess Detector (PHPMD) static analysis tool.
+
+### Installation
+
+1. If you haven't already, [install the Code Climate CLI](https://github.com/codeclimate/codeclimate).
+2. Run `codeclimate engines:enable phpmd`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
+3. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
+
+###Config Options
+
+Format the values for these config options per the [PHP_CodeSniffer documentation](https://github.com/squizlabs/PHP_CodeSniffer).
+
+* file_extensions - This is where you can configure the file extensions for the files that you want PHP_CodeSniffer to analyze.
+* rulesets - This is the list of rulesets that you want PHPMD to use while analyzing your files.
+
+###Sample Config
+
+    exclude_paths:
+     - "/examples/**/*"
+    engines:
+      phpmd:
+        enabled: true
+        config:
+          file_extensions: "php"
+          rulesets: "unusedcode"
```

Comment:
Similarly, my gut is we should allow either a single ruleset like this (`ruleset: unusedcode`) as a one-liner, or:

```yaml
rulesets:
  - unusedcode
  - design
  - naming
```

The engine can implode the array.